build.gradle

```diff
@@ -1,20 +1,18 @@
-
 buildscript {
-	repositories {
-		mavenCentral()
-		jcenter()
-	}
-
-	dependencies {
-		classpath 'net.saliman:gradle-cobertura-plugin:2.2.3'
-		classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.2'
+    repositories {
+        mavenCentral()
+        jcenter()
+    }
+
+    dependencies {
+        classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.2'
         classpath 'org.asciidoctor:asciidoctor-gradle-plugin:1.5.3'
         classpath 'org.ajoberstar:gradle-git:1.4.2'
-	}
+    }
 }
 
 apply plugin: 'groovy'
-apply plugin: 'cobertura'
+apply plugin: 'jacoco'
```

Comment:
Switching to jacoco from cobertura. A while back jacoco was causing issues with compilation of groovy, that appears to be fixed now.